src/interface/metadata.hpp

```diff
@@ -375,6 +375,9 @@ class Metadata {
   std::shared_ptr<Metadata> flux_metadata;
 
  public:
+  // TODO(JMM): I'm not sure where this should go...
+  static constexpr const std::size_t SWARM_DEFAULT_NMAX_POOL = 3;
+
```

Comment:
Yeah, good question. Eventually, we could consider separating concerns between dense, sparse, swarms, etc. but, on the other hand, there's also quite some overlap.
I think it's fine to keep it here for now.